src/kernel/include/inv.h

```diff
@@ -90,6 +90,26 @@ sret_deactivate(struct cap_captbl *t, capid_t capin, livenessid_t lid)
 { return cap_capdeactivate(t, capin, CAP_SRET, lid); }
 
 static int
+asnd_construct(struct cap_asnd *asndc, struct cap_arcv *arcvc, capid_t rcv_cap, u32_t budget, u32_t period)
```

Comment:
Did you ignore my previous comment about using this within the normal `asnd_activate` routine, or am I missing where it is used here?  This is code duplication right?  We don't like that if I'm right.